reports/_outliers.qmd

```diff
@@ -59,6 +59,12 @@ price_outlier_type %>%
 ## Outlier Types
 
 ```{r _outliers_type_breakdown}
+y_lim_axis_outlier_breakdown <- training_data %>%
+  filter(!sv_outlier_type %in% c("Not outlier", NA)) %>%
+  count(sv_outlier_type) %>%
+  summarise(max_value = max(n)) %>%
+  pull(max_value)
+
```

Comment:
I didn't realize @wrridgeway had already edited these to manually increase the y axis limits. I think this might be a small improvement still. The y axis is automatically scaled to make room for the highest value that is being visualized, stretched by a small additional amount to make room for labels on each bar.